site/content/contribute/getting-started/code-review.md

```diff
@@ -54,7 +54,7 @@ If you are a core committer seeking a review
 4. Assign a [QA tester](/contribute/getting-started/core-committers/#qa-testers) and label your pull request with `2: QA Review`.
     * It is the PM or QA tester's responsibility to determine the scope of required testing, if any.
     * Reviews by QA may occur at the same time as review by core committers. Be sure to ask for a second review as needed when changes are made.
-    * The QA tester should remove `2: QA Review` when their review is done, or if a review is deemed not necessary.
+    * The QA tester should add `QA Review Done` when their review is done, or if a review is deemed not necessary.
```

Comment:
We'll still have to remove `2: QA Review`, right?

Reply:
Oh my gosh! Yes, of course :) Let me fix these...

Reply:
Fixed!